### PR TITLE
document manageJobsNamespaceSelector

### PR DESCRIPTION
**Pull request by dgrove-oss**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Document how to use `manageJobsNamespaceSelector` in conjunction with `manageJobsWithoutQueueName`

#### Which issue(s) this PR fixes:

Part of #3589. 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by dgrove-oss** [comment]
I created a grouping for this topic on the web pages because I think it would make sense for the documentation for #2936 to be added as a a third file in site/content/en/docs/tasks/manage/enforce_job_management/.

/cc @yaroslava-serdiuk

File: site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Setup manageJobsWithoutQueueName"
+date: 2024-06-28
+weight: 10
+description: >
+  Using manageJobsWithoutQueueName and mangedJobsNamespaceSelector to prevent admission of Workloads without assigned LocalQueues.
+---
+
+This page describes how to configure Kueue to ensure that all Workloads submitted in namespaces
+intended for use by _batch users_ will be managed by Kueue even if they lack a local-queue label.
```
Comment (mimowo) [inline site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md:10]:
```suggestion
intended for use by _batch users_ will be managed by Kueue even if they lack the queue-name label.
```
maybe we also can give a useful link

File: site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+`managedJobsNamespaceSelector` is a Beta feature that is enabled by default.
+
+You can disable it by setting the `ManagedJobsNamespaceSelector` feature gate. Check the [Installation](/docs/installation/#change-the-feature-gates-configuration) guide for details on feature gate configuration.
+{{% /alert %}}
+
+## Expected Behavior
+
+In all namespaces that match the namespace selector, any Workloads submitted without a queue-name
+label will be suspended.  These Workloads will not be considered for admission by Kueue unless
+they are edited to have a queue-name label.
```
Comment (mimowo) [inline site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md:55]: ### 
```suggestion
label will be suspended.  These Workloads will not be considered for admission by Kueue until
they are edited to have the queue-name label.
```
nit, wdyt?

**Review by mimowo — COMMENTED** [review]
LGTM, just nits

File: site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md (hunk 1 above)
Comment (dgrove-oss) [reply site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md]: I changed to use the actual label not a shorthand.

**Comment by mimowo** [comment]
/lgtm
/approve
Once we make the code changes to support the selector for pods, we can follow up with docs update also [here](https://kueue.sigs.k8s.io/docs/tasks/run/plain_pods/).

File: site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md (hunk 1 above)
Comment (mimowo) [reply site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md]: fine

**Pull request merged**